### PR TITLE
disable operationOutcomes for the bulkdata feature

### PR DESCRIPTION
**Pull request by lmsurpre**
due to https://github.com/IBM/FHIR/issues/2910

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: charts/ibm-fhir-server/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Helm chart for the IBM FHIR Server
 name: ibm-fhir-server
-version: 0.4.2
+version: 0.4.3
```
Comment (michaelwschroeder) [inline charts/ibm-fhir-server/Chart.yaml:4]: should also update `artifacthub.io/changes` on lines 30-31

**Review by michaelwschroeder — APPROVED** [review]
LGTM - just the one comment in Chart.yaml

File: charts/ibm-fhir-server/Chart.yaml (hunk 1 above)
Comment (lmsurpre) [reply charts/ibm-fhir-server/Chart.yaml:4]: done

**Comment by lmsurpre** [comment]
note:  i merged the extraVolumes change into this one, so now 0.4.3 will contain both of these changes

**Pull request merged**